__tests__/lib/string.test.ts

```diff
@@ -1,11 +1,11 @@
 import {RichText} from '@atproto/api'
 
-import {parseEmbedPlayerFromUrl} from 'lib/strings/embed-player'
+import {parseEmbedPlayerFromUrl} from '#/lib/strings/embed-player'
```

Comment:
```suggestion
import {parseEmbedPlayerFromUrl} from 'lib/strings/embed-player'
```

Reply:
The imports are correct, there's now a lint error that enforces the #/ prefix for alias imports

Reply:
thanks for your patient explanation